### DISCUSSION TITLE
Deprecation notice: Asset

### DISCUSSION THREAD

**Discussion by mdt2** [adobe/spectrum-css#2539] [Announcements]
As of Feb 16, 2024, @spectrum-css/asset has been deprecated.

This component has been removed from the Spectrum CSS monorepo and will no longer receive new features. Bug fixes will be patched on a as-needed basis. If you find a critical bug, please open a bug report.

Bug fixes will no longer be supported after the project migrates fully to Spectrum 2 (date TBD).

If you have any questions or concerns about this deprecation, please feel free to contribute to the discussion here or reach out to us directly.

**Comment by castastrophe** [comment]
Additional note: this component has been migrated to the new token system and is not going to be removed quite yet. Still don't recommend using it in new implementations though.
